src/material/form-field/form-field-control.ts

```diff
@@ -30,7 +30,7 @@ export abstract class MatFormFieldControl<T> {
   readonly placeholder: string;
 
   /** Gets the NgControl for this control. */
-  readonly ngControl: NgControl | null;
+  readonly ngControl: NgControl | AbstractControlDirective | null;
```

Comment:
since `NgControl` is a subclass of `AbstractControlDirective`, do we even need to leave it in the list?

Reply:
Hmm, good point. I guess we don't since we only access fields that are on both of them. Changing...